site/en/blog/fedcm-origin-trial/index.md

```diff
@@ -554,102 +571,44 @@ The browser expects a JSON response that includes the following property:
     </tr>
   </thead>
   <tr>
-    <td><code>id_token</code> (required)</td>
-    <td><a href="https://openid.net/specs/openid-connect-core-1_0.html#IDToken"><code>IDToken</code></a> is a security token (string) that contains claims about the authentication.</td>
+    <td><code>token</code> (required)</td>
+    <td>A token is a security token (string) that contains claims about the authentication.</td>
```

Comment:
This doesn't sound very good, "a token is a security token" sounds almost like "a token is a token". 

Could we just edit to "A token is a string that contains claims about the authentication"?

Reply:
Yeah, that sounds better.